app/services/injection_response_service.rb

```diff
@@ -1,14 +1,14 @@
 class InjectionResponseService
   def initialize(json)
-    @response = json.stringify_keys
-    raise ParseError, 'Invalid JSON string' unless @response.keys.sort.eql?(%w[errors from messages uuid])
-    @claim = Claim::BaseClaim.find_by(uuid: @response['uuid'])
+    @response = json.symbolize_keys
+    raise ParseError, 'Invalid JSON string' unless @response.keys.sort.eql?(%i[errors from messages uuid])
+    @claim = Claim::BaseClaim.find_by(uuid: @response[:uuid])
     @channel = Claims::InjectionChannel.for(@claim)
   end
 
   def run!
-    slack.build_injection_payload(@response)
-    return failure(action: 'run!', uuid: @response['uuid']) unless @claim
+    slack.build_payload(**@response)
```

Comment:
nit - `notifier` or `slack_notifier` probably a clearer name since `slack` is the class name